test/runnable/complex.d

```diff
@@ -338,6 +339,17 @@ void test7976() {
     auto b = a[0] = a[1];
 }
 
+/***************************************/
+// https://issues.dlang.org/show_bug.cgi?id=20688
+
+void test20688()
+{
+    cdouble r = cpowf(2. + 0i, 2. + 0i);
+    assert(cast(double)creal(r) == 4.0);
+    assert(cast(double)cimag(r) == 0.0);
+    printf("%f+%fi\n", cast(double)creal(r), cast(double)cimag(r));
```

Comment:
You can use `RUN_OUTPUT` to check the output for correctness.

Reply:
Or remove the printf as we're already covered by asserts?

Reply:
The printf is nice to save typing when it doesn't work.